assets/sass/v2/_footer.scss

```diff
@@ -4,7 +4,7 @@
     display: flex;
     flex-direction: column;
     .link {
-      width: max-content;
+      width: fit-content;
```

Comment:
Another possible regression, your changes here still good? https://github.com/okta/okta-signin-widget/pull/1904

Reply:
yeah i verified that first :)